anatomy/tracks/concept-exercises.md

```diff
@@ -72,6 +72,19 @@ exercises
         └── CarsAssemblyTests.cs (tests)
 </pre>
 
+### Minimal valid spec
+
+We favor an "optimistic merging" approach to new exercises, where tracks can develop exercises in a "work in progress" state.
+The minimal valid state, which will pass configlet and allow you to merge is:
+- Valid entry in the track `config.json`, with the `status` set to `wip`.
```

Comment:
By this we mean that all the linting rules defined for concept exercise entries in the config.json should pass? I think we  probably need to skip some checks:

- The `files.exemplar` entry should be allowed to be empty (which the spec says it can't be)
- Checking whether the `concepts` and `prerequisites` have corresponding `config.json` entries would be good to have, but this would then also imply having files for those concepts (`concepts/<slug>/introduction.md` etc.). Do we want that?

One consequence of this would be that the concepts and prerequisites would have to have corresponding

Reply:
> Checking whether the concepts and prerequisites have corresponding config.json entries would be good to have, but this would then also imply having files for those concepts (concepts/<slug>/introduction.md etc.). Do we want that?

I would say we do.  I can only speak for my track, but I have frequently run into issues where concepts have been broken down, combined, or (_often_) misspelled/paraphrased and their corresponding references or files either don't match or haven't been updated.  

Looking at the directory in isolation doesn't give any hints that this happened ... and reviewing the exercise and track `config.json` files as a human doesn't always catch the problem.   If a check errored out, that would be a flag that either a required file is missing/needs to be filled in -- or that there is a file/concept naming issue.

Reply:
To tag onto that:  I would want it to be the same behavior as for the exercise files -- the _stub_ needs to be in place, and the `config.json` entries need to match -- but the files themselves can be empty (_which would prompt a warning_).